lib/v2/screens/settings/components/settings_card.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/services/navigation_service.dart';
+
+/// SETTINGS CARD
+class SettingsCard extends StatelessWidget {
+  /// Card icon
+  final IconData icon;
+
+  /// The text title in the first row
+  final String title;
+
+  /// The text value next to the title
+  final String titleValue;
+
+  /// To set a different color to titleValue
+  final Color titleValueColor;
```

Comment:
do we need this?
Looking at the designs all i see is white title color?

Reply:
???

Reply:
Im waiting for the @JGDoh  response.